lib/ui/datepicker/datepicker.js

```diff
@@ -199,7 +199,8 @@
 
         scope.$evalAsync(function() {
           element.datepicker(avDatepicker.options);
-      });
+          element.inputmask('99/99/9999');
```

Comment:
* the format is hardcoded
* there should be a way to disable the mask if not needed
* the mask probably needs to belong in another directive